README.md

```diff
@@ -61,3 +61,52 @@ http://localhost:5000/username
 ```
 to delete a user account.
 You can pass the relavant parameters in Postman API Client `body` tab. 
+
+## Expected environment variables
+This server setup expects the following environment variables for sucessful
+setup of the server and the MongoDB database. Make sure you create a `.env` file
+at the root of project directory and include the following variables in the env
+file. The following are defult values. Don't forget to set the variables to
+approprite values.
+
+ - DB_HOST=127.0.0.1
+ - DB_PORT=27017
+ - DB_NAME=i3wd
+ - DB_USER=root
+ - DB_PASSWORD=rootpassword
+ - PORT_BACKEND=5000
+ - SESSION_TOKEN_SECRET_KEY=mysecretkey
+ - SESSION_TOKEN_DURATION=1h
+ - SESSION_TOKEN_REFRESH_SECRET_KEY=refreshsecretkey
+
+## How to run this server
+Assuming you use the above default values, 
+  1. To start the database, execute the following 
+  ```bash
+  sudo docker-compose up -d
+  ```
+  at the root of the project directory.
+
+  2. To start the server, execute the following 
+  ```bash
+  npm run devStart
+  ```
+  at the root of the project directory.
+
+  3. To establish an interactive connection to the database using mongosh, execute
+  ```bash
+  sudo docker exec -it mongodb mongosh -u root -p rootpassword --authenticationDatabase admin 
```

Comment:
Nit: It´s consistent in the documentation, but maybe is not obvious that user and password need to correspond to the values of the environment variables DB_USER and DB_PASSWORD.
```suggestion
  sudo docker exec -it mongodb mongosh -u <DB_USER> -p <DB_PASSWORD> --authenticationDatabase admin 
```